Makefile

```diff
@@ -100,7 +101,7 @@ rpm-tarball:
 	    | gzip > k4e-agent-$(VERSION).tar.gz
 
 rpm-src: rpm-tarball
-	cp k4e-agent-$(VERSION).tar.gz $(HOME)/rpmbuild/SOURCES/
+	install -D -m644 k4e-agent-$(VERSION).tar.gz --target-directory `rpmbuild -E %_sourcedir`
```

Comment:
I'm ok with that, if you run make rpm-copr, this will work?

here is how it works:
https://github.com/jakub-dzon/k4e-device-worker/blob/main/.github/workflows/rpm.yaml

Reply:
Yep, it builds the rpm files and this [line](https://github.com/jakub-dzon/k4e-device-worker/blob/main/.github/workflows/rpm.yaml#L30) is no longer necessary.


```bash
[root@83a558d5e6fa k4e-device-worker]# rm -Rf ~/rpmbuild/
[root@83a558d5e6fa k4e-device-worker]# make rpm-tarball
which: no ginkgo in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
which: no gover in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
(git archive --prefix k4e-agent-1.0/ HEAD ) \
    | gzip > k4e-agent-1.0.tar.gz
[root@83a558d5e6fa k4e-device-worker]# make rpm-src
which: no ginkgo in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
which: no gover in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
(git archive --prefix k4e-agent-1.0/ HEAD ) \
    | gzip > k4e-agent-1.0.tar.gz
install -D -m644 k4e-agent-1.0.tar.gz --target-directory `rpmbuild -E %_sourcedir`
rpmbuild -bs \
	-D "VERSION 1.0" \
	-D "RELEASE 1" \
	-D "_libexecdir /usr/libexec" \
	--buildroot /root/k4e-device-worker/dist ./k4e-agent.spec
Wrote: /root/rpmbuild/SRPMS/k4e-agent-1.0-1.el8.src.rpm
```

I did not succeed in running the last command because I could not find the `copr-cli` package in CentOS. But I wager that it does not matter at that point since the `.src.rpm` file is already in place.

Reply:
Then works for me :-) 

Thanks jordi